language/move-prover/tests/sources/functional/verify_vector.move

```diff
@@ -231,21 +252,41 @@ module VerifyVector {
 
     // Remove the `i`th element E of the vector, shifting all subsequent elements
     // It is O(n) and preserves ordering
-    fun verify_remove<Element>(v: &mut vector<Element>, i: u64): Element {
-        let len = Vector::length(v);
+    fun verify_remove<Element>(v: &mut vector<Element>, j: u64): Element {
+        let vlen = Vector::length(v);
```

Comment:
Just a note for later. We may need to adjust the name changes (i.e., "len" -> "vlen", and "i" -> "j") later when we add these specs into the actual stdlib vector file (`language/stdlib/modules/vector.move`).

Reply:
There is a related bug on using len #4355 .